bin/compose-up-simulator.sh

```diff
@@ -11,6 +11,8 @@ command -v az >/dev/null 2>&1 || {
     exit 1
 }
 
+az acr login -n hmctspublic
```

Comment:
you shouldn't need to log in to hmcts public. If you get an authentication error you can fix it with a docker logout hmctspublic.azurecr.io